docs/release.md

```diff
@@ -0,0 +1,55 @@
+# Status AddOn Relesea process 
+
+The Status AddOn release process is based on [GoReleaser](https://goreleaser.com).
+GoReleaser is configured to automatically create a new release by pushing new tags matching the
+pattern 'v*' and build and publish to ghcr.io the packages for the 
+[container image](https://github.com/kubestellar/ocm-status-addon/pkgs/container/ocm-status-addon)
+and the [helm chart](https://github.com/kubestellar/ocm-status-addon/pkgs/container/ocm-status-addon-chart) 
+used to deploy the status add-on. The relevant files for goreleaser configuration are
+[./goreleaser.yaml](../.goreleaser.yaml) and [./.github/workflows/goreleaser.yml](../.github/workflows/goreleaser.yml).
+
+The steps outlined below assume that a release branch for the release that is going to be
+created already exists. Typically a release branch is created at the beginning of each
+new release cycle with `git checkout -b <release branch>`.
+
+## Steps to make release
+
+1. Fetch from upstream and checkout main:
+```shell
+git fetch upstream
+git checkout main
+```
+2. Rebase into main
+```shell
+git rebase upstream/main
```

Comment:
Something is confused here. Nobody does `git checkout main; git rebase upstream/main`. I might do `git checkout main; git merge --ff-only upstream/main`, when I am keeping my own `main` branch updated by `upstream/main`. But none of this is about creating a release.

Reply:
Updated based on proposal below.